quickstarts/angular1/01-login.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Setup and Authentication
+name: Angularjs-login
+alias:
+  - angular
+  - angularjs
+language:
+  - Javascript
+framework:
+  - AngularJS
+image: /media/platforms/angular.png
+tags:
+  - quickstart
+  - angular
+snippets:
+  dependencies: /quickstarts/angular1/snippets/dependencies
+  init: /quickstarts/angular1/snippets/init
+  head: /quickstarts/angular1/snippets/head
+alias:
+  - angular
+---
+
+## Setup and Authentication
+
+One of the most aching aspects of an Angular App is authentication. It becomes worse when you have routes to protect, page refresh to deal with, tokens to manage, multi-factor authentication, and so on. This quickstart is planned to help you tackle these challenges.
+
+This is the very beginning of a simple, practical and multi-step quickstart that will guide you through managing authentication in your Angular JS apps with Auth0.
+
+
+### Create an Application
+
+Create an [Auth0 account](https://manage.auth0.com/login) and an authentication application from the [dashboard](https://manage.auth0.com/#/). Once you create an app, you'll be provided with credentials (Domain, Client ID, and Client Secret) which should be stored somewhere safe (do not commit this information to your git repo!).
+
+![App Dashboard](/media/articles/angularjs/app_dashboard.png)
+
+TODO: Find out how to configure angular-sample URL
+<%= include('../_includes/_package', {
+  pkgRepo: 'auth0-angular',
+  pkgBranch: 'master',
+  pkgPath: 'examples/widget-redirect',
+  pkgFilePath: null,
+  pkgType: 'js'
+}) %>_
+
+::: panel-info Running the Sample
+At any point in time, you can run the sample with `http-server`. First install with `npm install -g http-server` then run `http-server` on the root directory of the sample.
+:::
+
+::: panel-info Default Configuration
+Every instance of the seed project comes configured with your `Default App` credentials. Awesome right?
+:::
+
+### Configure Callback URLs
+
+Callback URLs are URLs that Auth0 invokes after the authentication process. It is especially useful in OAuth. Callback URLs can be manipulated on the fly and that could be harmful.
+
```

Comment:
Remove this " It is especially useful in OAuth."

Move and rephrase this "Callback URLs can be manipulated on the fly and that could be harmful." to the following paragraph. Just a single paragraph about security.